### ISSUE TITLE
[Questions] Is the AltStore repo still active and viable for installing?

### ISSUE THREAD

**Issue by blakegearin** [qnblackcat/uYouPlus#1538]
### Have you read the FAQ?

- [X] Yes, I did read the FAQ

### Is there an existing issue/question for this?

- [X] I have searched the existing issues

### Do you think this is a bug?

✅ Yes, I believe this is a bug. I will open a new issue with the bug template

### My question

From my attempts, it seems not. If that's the case, it'd probably be best to remove references in the documentation to avoid confusing new users trying to install.

**Comment by ShadowJonathan** [comment]
What error do you get? Is it "this app is not notarized" or the likes?

**Comment by DandeMC** [comment]
Yes, I become this message and I cant add the source. What can I do?

**Comment by otherguy** [comment]
I can add the repo just fine, but installing the latest version fails with "The file doesn't exist". This makes sense, because the JSON file references the file https://github.com/qnblackcat/uYouPlus/releases/download/v19.08.2-3.0.3/uYouPlus-19.08.2-3.0.3.ipa which doesn't exist.

**Comment by hnguyen1910** [comment]
> I can add the repo just fine, but installing the latest version fails with "The file doesn't exist". This makes sense, because the JSON file references the file https://github.com/qnblackcat/uYouPlus/releases/download/v19.08.2-3.0.3/uYouPlus-19.08.2-3.0.3.ipa which doesn't exist.

All the releases are removed by the Google DMCA so that’s why it doesn’t exist. To use you must build the ipas yourself.
